mlflow/models/evaluation/base.py

```diff
@@ -775,7 +775,18 @@ def _start_run_or_reuse_active_run():
         yield active_run.info.run_id
 
 
-def _resolve_default_evaluator(model_type, evaluator_config):
+# NB: We often pass around evaluator name, config, and its instance together. Ideally, the
```

Comment:
note: moved up the code block to use `EvaluatorBundle` in the type annotation of `_resolve_default_evaluator`.